modules/ROOT/pages/configure-global-edifact-receive-settings.adoc

```diff
@@ -1,6 +1,6 @@
 = Configuring Global EDIFACT Receive Settings
 
-When EDIFACT messages received from your trading partners do not contain a UNA segment, Partner Manager uses the delimiters specified in the *Global EDIFACT settings: Receive from partners* window to parse the received EDIFACT payloads. These settings are defined under the host organization’s profile:
+When EDIFACT messages received from your partners do not contain a UNA segment, Partner Manager uses the delimiters specified in the *Global EDIFACT settings: Receive from partners* window to parse the received EDIFACT payloads. These settings are defined under the host organization’s profile:
```

Comment:
When EDIFACT messages received from your partners don't contain a UNA segment, Partner Manager uses the delimiters specified in the *Global EDIFACT settings: Receive from partners* window to parse the received EDIFACT payloads. You define these settings under the host organization’s profile: